### ISSUE TITLE
UNK tag decrease f1-score

### ISSUE THREAD

**Issue by NicolasVautier** [flairNLP/flair#2761]
Hello, this is my first time posting ever i will try to be as precise as possible.

i'm training a SequenceTagger model using flair 0.11.2 with 3 tags and i'm having issues with the unwanted add of unk tag to my tag dict which decreases micro f1-score (is it a normal behaviour ?). I've seen other issues about the unk taf but none speaks about f1-score.

Here is my code : 

            columns = {0 : "text", 1 : "ner"}
            corpus: Corpus = ColumnCorpus(path, columns,
                                        train_file = path + '/train.txt',
                                        test_file = path + '/test.txt')
            
            label_type = 'ner'

            tag_dictionary = corpus.make_label_dictionary(label_type=label_type)

            embeddings = StackedEmbeddings([WordEmbeddings('glove')])

            tagger = SequenceTagger(hidden_size=128,
                                                embeddings=embeddings,
                                                tag_dictionary=tag_dictionary,
                                                tag_type=label_type,
                                                use_crf=True)

            trainer = ModelTrainer(tagger, corpus)

            # Start training
            trainer.train(path,
                        learning_rate=0.1,
                        mini_batch_size=32,
                        max_epochs=10)

But when creating label_dictionary <unk> label is added and taken into account for training wich gives low f1-score micro avg despite good overall f1-score for other labels

![image](https://user-images.githubusercontent.com/48012106/168046335-192f5458-0092-4390-b1f4-7b49a522f3d1.png)

A quick fix would be to use the remove_item function of Dictionary class but it seems to dramatically decrease f1-score for other classes, maybe exclude unk from f1 score or add the possibility to edit add_unk directly when calling make_label_dictionary ?

Ps : my train.txt and test.txt files looks like this
`
word1 O
word2 O
word3 attributeA00356
word4 O
word5 attributeA00436
word6 O
word7 attributeA00354

**Comment by djstrong** [comment]
It looks like micro f1 is not correctly calculated.
